Dockerfile

```diff
@@ -40,6 +47,9 @@ RUN apk-install git && \
 RUN cp /logstash-1.5.1/vendor/bundle/jruby/1.9/gems/*/vendor/jar-dependencies/runtime-jars/log4j-1.2.17.jar \
        /logstash-1.5.1/vendor/bundle/jruby/1.9/gems/logstash-output-elasticsearch*/vendor/jar-dependencies/runtime-jars/
 
+RUN apk-install ca-certificates
+ENV SSL_CERT_FILE /etc/ssl/certs/ca-certificates.crt
```

Comment:
If you set `SSL_CERT_FILE` here, I think it will override the root certificates OpenSSL uses in the syslog output that we're explicitly installing here: https://github.com/aptible/gentlemanjerry/blob/e66cfbf254e52abd2a9a0beb0d274ea71c28fc63/Dockerfile#L5-L22. We do that root certificate override for either papertrail and/or logentries, at least one of which didn't work with the default ca-certificates package in debian. The last time I checked was probably a year ago, so this could very well have changed since then.

I definitely like running `apk-install ca-certificates` better if that works, we'd just need to know that the resulting gentlemanjerry build could still send logs over TLS to papertrail and logentries without certificate errors. If that works, then the download and keytool install nonsense could just be removed from the Dockerfile in favor of these two lines that you've added, which would be a big simplication and improvement.

IIRC, papertrail should work fine but logentries has issues - see this page where they encourage "cut'n pasting" of their certificates to make it all work: https://logentries.com/doc/certificates/